src/wp-includes/rest-api/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -1681,7 +1681,7 @@ public function get_collection_params() {
 			'default'           => 'approve',
 			'description'       => __( 'Limit result set to comments assigned a specific status. Requires authorization.' ),
 			'sanitize_callback' => 'sanitize_key',
```

Comment:
This is an issue since it only does anything for strings.

Reply:
Actually @TimothyBJacobs I'm not sure we need this - 
* sanitization only called when creating or updating a comment - in this case we should still only accept a single status value, right? sanitize_key would throw an error, maybe thats what we want?

Also, while writing tests, I noticed as far as I can see we already strictly enforce strict status values from a list in `WP_Test_REST_Comments_Controller::handle_status_param`. Should the endpoint accept custom status values like `WP_Comment_Query`?

Reply:
Thanks for the feedback @TimothyBJacobs 

We don't need this immediately so I am deprioritizing it for the moment.